test/integration_tests/src/test_prepared.cpp

```diff
@@ -370,7 +338,7 @@ BOOST_AUTO_TEST_CASE(massive_number_of_prepares)
   test_utils::execute_query(session, create_table_query);
 
 
-  CONTAINER<boost::unique_future<CassPreparedMovable> > prepare_futures;
+  CONTAINER<boost::unique_future<test_utils::CassPreparedPtr> > prepare_futures;
```

Comment:
Does not build locally for me, using boost 1_55. Not sure if it warrants a change